MLIntegration/ios/App/App/config.xml

```diff
@@ -2,5 +2,9 @@
 <widget version="1.0.0" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
   <access origin="*" />
   
+  <feature name="Device">
+    <param name="ios-package" value="CDVDevice"/>
+  </feature>
+
   
 </widget>
```

Comment:
same

Reply:
@jesusdiez This is a self-generated file by ionic too.